### ISSUE TITLE
Custom build not including some requested tags

### ISSUE THREAD

**Issue by daamsie** [mattiasw/ExifReader#126]
### Description

I'm trying to generate a custom build to reduce my bundle size. I managed to create the custom build described in the read me just fine, but when I try to add other tags that I need access to, I can't seem to get it right. I've dug right into the TagFilterPlugin.js file that seems to handle this and it just is not including the tags I want. 

These are the relevant tags that I need in my output

```
"DateTime",
"DateTimeOriginal",
"Headline",
"Orientation",
"ImageDescription",
"Caption/Abstract",
"GPSLatitude",
"GPSLatitudeRef",
"GPSLongitude",
"GPSLongitudeRef"
```

**How to reproduce**

I updated my package.json to include the following (also unclear on how much to include in exif, xmp, iptc, so just threw it in all of them)
```

"exifreader": {
      "include": {
          "jpeg": true,
          "png": true,
          "file": true,
          "png_file": true,
          "exif": [
            "DateTime",
            "DateTimeOriginal",
            "Headline",
            "Orientation",
            "ImageDescription",
            "Caption/Abstract",
            "GPSLatitude",
            "GPSLatitudeRef",
            "GPSLongitude",
            "GPSLongitudeRef"
          ],
          "xmp": [
            "DateTime",
            "DateTimeOriginal",
            "Headline",
            "Orientation",
            "ImageDescription",
            "Caption/Abstract",
            "GPSLatitude",
            "GPSLatitudeRef",
            "GPSLongitude",
            "GPSLongitudeRef"
          ],
          "iptc": [
            "DateTime",
            "DateTimeOriginal",
            "Headline",
            "Orientation",
            "ImageDescription",
            "Caption/Abstract",
            "GPSLatitude",
            "GPSLatitudeRef",
            "GPSLongitude",
            "GPSLongitudeRef"
        ]
      }
  },
```

**What I expected would happen:**

I expected to be able to read the Orientation of images

**What really happened:**

No orientation to be seen. 

Not sure if I'm doing something wrong, or there's an issue in the build script here. At the very least it maybe can be made clearer in the documentation.

**Comment by mattiasw** [comment]
Hi! I will have a look sometime during the weekend. I don't see any obvious errors in your setup, but two things that will be my first to check is:
- The "Caption/Abstract" tag. The slash might do something strange while parsing. You could try removing that specific tag to check.
- Having tags that don't belong in that group. This feels less likely to cause an issue though. But if you pass in `{expanded: true}` (like so: `ExifReader.load(fileBuffer, {expanded: true})`) to a non-optimized build of the library the tags will be grouped where they belong. Then specify them in the right place in `package.json`.

If you have time to try them out and it worked, please tell me and I'll try to make a bugfix for it. I don't think you're doing anything wrong. :-)

**Comment by daamsie** [comment]
Thanks for the response. No joy, but I tried reducing my test down to just exif (where orientation was for my test image) and the problem still exists. Using the current package.json settings: 

`"exifreader": {
      "include": {
          "jpeg": true,
          "png": true,
          "file": true,
          "png_file": true,
          "exif": [
            "DateTime",
            "DateTimeOriginal",
            "Orientation",
            "GPSLatitude",
            "GPSLatitudeRef",
            "GPSLongitude",
            "GPSLongitudeRef"
          ]
      }
  }`

running with expanded:true, I get the following for exif (this file has no lat/lng, so no surprises it's missing below): 

- DateTime
- DateTimeOriginal
- Exif IFD Pointer
- undefined-271
- undefined-272
- etc.. (lots more undefined keys that probably don't need to be returned at all)

**Comment by mattiasw** [comment]
Fixed in version 3.12.3. It affected several tags actually. Big thanks for pointing it out!

**Comment by daamsie** [comment]
Excellent. Thank you!
